rust/signed_doc/tests/common/contest_ballot_checkpoint.rs

```diff
@@ -0,0 +1,34 @@
+use catalyst_signed_doc::providers::tests::TestCatalystProvider;
+use ed25519_dalek::ed25519::signature::Signer;
+
+use super::*;
+
+/// Creates a contest ballot checkpoint document.
+pub fn contest_ballot_checkpoint_doc(
+    linked_doc: &CatalystSignedDocument,
+    parameters_doc: &CatalystSignedDocument,
+    provider: &mut TestCatalystProvider,
+) -> anyhow::Result<CatalystSignedDocument> {
+    let id = UuidV7::new();
+    let (sk, kid) = create_dummy_key_pair(None);
+    provider.add_sk(kid.clone(), sk.clone());
+
+    let linked_ref = linked_doc.doc_ref()?;
+    let parameters_ref = parameters_doc.doc_ref()?;
+    let chain = Chain::new(0, None);
+
+    Builder::new()
+        .with_json_metadata(serde_json::json!({
+            "content-type": ContentType::Cbor,
+            "content-encoding": ContentEncoding::Brotli,
+            "type": doc_types::CONTEST_BALLOT_CHECKPOINT.clone(),
+            "id": id,
+            "ver": id,
+            "ref": linked_ref,
```

Comment:
It should be an array no ?  The `ref` field is a `DocumentRefs` type which is an array of documents
```suggestion
            "ref": [linked_ref],
```